File: DrawerTestApp/app/src/main/java/com/redenvy/drawertestapp/MainActivity.java
package com.redenvy.drawertestapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Switch;
import android.widget.Toast;

import com.redenvy.drawertestapp.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {
private ActivityMainBinding binder;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binder = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binder.getRoot());
        binder.popup.setOnClickListener(this);
        binder.drawer.setOnClickListener(this);
        binder.img.setOnClickListener(this);
        binder.tabs.setOnClickListener(this);
        binder.trans.setOnClickListener(this);
    }
    @Override
    public void onClick(View view){
        switch (view.getId()){
            case R.id.popup:
                startActivity(new Intent(MainActivity.this, Popup.class));
                break;
            case R.id.drawer:
                startActivity(new Intent(MainActivity.this, drawer.class));
                break;
            case R.id.img:
                startActivity(new Intent(MainActivity.this, images.class));
                break;
            case R.id.tabs:
                startActivity(new Intent(MainActivity.this, tabs.class));
                break;
            case R.id.trans:
                startActivity(new Intent(MainActivity.this, transition.class));
                break;
            default:
                break;
        }
    }
}

File: JustDoIt/app/src/main/java/com/redenvy/justdoit/ui/MainActivity.kt
package com.redenvy.justdoit.ui

import android.content.Intent
import android.content.SharedPreferences
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.activity.viewModels
import androidx.appcompat.app.AppCompatDelegate
import androidx.lifecycle.ViewModel
import com.redenvy.justdoit.R
import com.redenvy.justdoit.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    private val viewModel: ViewModel by viewModels()
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        binding.apply {
            settingsIcon.setOnClickListener(){
                startActivity(Intent(applicationContext,SettingsActivity::class.java))
            }
        }
        apiCall()
    }
    private fun apiCall(){
        //TODO: find out what is supposed to be here.
    }
}